client/src/pages/serviceChange/index.js

```diff
@@ -24,8 +28,9 @@ function ServiceChange() {
 
       const response = await serviceChange({ signers, limitChange });
       console.log(`Received response: ${response.data}`);
-      navigate('/submitted?1');
+      navigate('/submitted');
```

Comment:
Each scenario has an individual description on the results page. The description for this scenario is determined by the presence of "?1" in the link. Please put "?1" back in the link.

Reply:
Fixed